frontend/crd/src/app/tasks-modal/tasks-modal.component.html

```diff
@@ -3,17 +3,25 @@
     <h1 id="modal-title"> {{ modalData.task.name }}</h1>
   </header>
   <div id="modal-body">
-    <div id="">
-      <p style="font-size:30px"><b>Description: </b><br>
-        {{ modalData.task.description }}</p>
+    <div class="box">
+      <div class="text">
+        <p class="header">Year Level:</p>
```

Comment:
maybe the header should just be the year level itself?

Reply:
also change this from p class="header" to something like h2-h4 or whatever. same goes for description